tests/display.rs

```diff
@@ -172,6 +180,33 @@ mod structs {
                     .to_string(),
                     "struct",
                 );
+                assert_eq!(
+                    format!(
+                        "{:?}",
+                        Struct {
+                            field1: 0,
+                            field2: 3
+                        }
+                    ),
+                    "Struct: 0 3"
+                );
+                assert_eq!(
+                    Struct2 {
+                        field1: 0,
+                        field2: 3
+                    }.to_string(),
+                    "Struct2"
+                );
+                assert_eq!(
+                    format!(
+                        "{:?}",
+                        Struct2 {
+                            field1: 0,
+                            field2: 3
+                        }
+                    ),
+                    "Struct2"
+                );
```

Comment:
Should these cases error?

Reply:
```rust
                assert_eq!(
                    Struct2 {
                        field1: 0,
                        field2: 3
                    }.to_string(),
                    "Struct2"
                );
```
@ilslv @tyranron do we want to reconsider this behaviour?

Reply:
The [described and implemented design](https://github.com/JelteF/derive_more/pull/225#issuecomment-1319892317) misses default multi-field struct case.

@ilslv should we error in this case?

Reply:
@JelteF @tyranron yes, this definitely should produce compile-time error.

Reply:
@ilslv please, make a fix in a separate PR. Thanks!

Reply:
Fixed in https://github.com/JelteF/derive_more/pull/243